src/webloader.js

```diff
@@ -44,11 +44,15 @@ function currentScript() {
 
 export function initialize(callback) {
 	// If the user has requested not to be tracked, or the browser is older than ie11, bail out.
-	if ((!navigator.doNotTrack || navigator.doNotTrack === 'unspecified' || navigator.doNotTrack === '0') && typeof Map !== 'undefined') {
+	if (typeof Map !== 'undefined') {
 		const script = currentScript();
 		const dataset =  ensureEnvironmentOnDataset(script);
 		const config = buildConfig(dataset);
 
+		if (config.dntSupport && (navigator.doNotTrack === '1' || window.doNotTrack === '1')) {
+			return;
+		}
+
```

Comment:
Thanks. I assume `navigator.doNotTrack === '1'` is the inverse of `navigator.doNotTrack === 'unspecified' || navigator.doNotTrack === '0'`, right?